pages/docs/ecosystem-roles/relayer/running-relayer/quick-start.mdx

```diff
@@ -106,16 +113,29 @@ export WEBB_PORT=9955
 
 For our example configuration file, the following environment variables are required:
 
-- `PRIVATE_KEY` - The private key of the relayer. This can be a hex-encoded private key or a mnemonic phrase.
+- `PRIVATE_KEY` - The private key of the relayer. This is the hex-encoded private key. Use of the mnemonic phrase is not currently supported.
 
-create a new file called `.env` in the same directory as your current directory and add the following line:
+Create a new file called `.env` in the same directory as your current directory and add the following line:
 
 ```sh filename="PRIVATE_KEY" copy
 # Hex-encoded private for the relayer (64 characters) prefixed with 0x
 PRIVATE_KEY="0x..."
 ```
 
-save the file and exit the editor.
+Save the file and exit the editor.
+
+Set the environment variable in current environment:
+
+```sh
+source ~/.config/webb-relayer/.env
+```
+
+Update your .bashrc or .zshrc file to set those environment variables each time you start a new terminal session:
+
+```sh
+echo 'source ~/.config/webb-relayer/.env' >> ~/.bashrc
+```
```

Comment:
This part is not necessary, as the relayer will try to find the `.env` file and load it for you.